src/components/submission/table-view.vue

```diff
@@ -101,7 +101,8 @@ const odataSelect = computed(() => {
 });
 
 // `clear` indicates whether this.odata should be cleared before sending the
-// request. `refresh` indicates whether the request is a background refresh.
+// request. `refresh` indicates whether the request is a background refresh
+// (whether the refresh button was pressed).
```

Comment:
This parenthetical existed in `EntityList`, so I thought I'd copy it here as well for consistency.